content/en/real_user_monitoring/funnel_analysis/_index.md

```diff
@@ -0,0 +1,102 @@
+
+When you have a starting point in mind, but aren't sure what your users did next, suggested next steps helps remove the guesswork. After inputting steps, the suggested next steps panel shows you the top five most common views that users typically have next, as well as the top five most common actions that users typically take next. This allows you to build funnels quicker knowing the paths your users are taking in sequence.
+
+{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-suggested-next-steps.jpg" alt="Build a funnel" style="width:90%;" >}}
+
+### Filtering
+
+When constructing your funnel, you can add all [default attributes][1] (core, device, operating system, geo-location, and user) and [session-specific][2] attributes to analyze the data further.
+
+{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-filtering.jpg" alt="Use filters to drill down information when constructing your funnel" style="width:50%;" >}}
+
+## Analyzing a funnel
+
+After building a funnel, to understand why you see the conversion rate you do, the Funnel Insights panel offers correlated data on performance and user behavior trends.
+
+For high level trends, you can see the end-to-end conversion rate for your entire workflow and also see individual steps to step conversion to dropoff rates. If you want to understand what it looks like for someone who converted versus someone who dropped off, you can watch a session replay for each case.
+
+{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-analyzing-funnel.jpg" alt="Use the Funnel Insights panel to review performance and user behavior trends" style="width:90%;" >}}
+
+The performance section allows you to understand if poor performance could have affected conversion. You can view a graph with a correlation between the load time of that page and the conversion rate and also see if any issues (detected by Error Tracking) occurred on that page.
+
+The user behavior section allows you to compare the average frustration count (from [frustration signals][3]) with the conversion rate, and even drill into the frustration signals detected from individual actions. Next to this section is a chart showing the conversion and dropoff rate for specific countries, allowing you to understand if geographic region plays a role in a user converting. 
+
+{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-user-behavior.jpg" alt="User behavior section within funnel analysis" style="width:90%;" >}}
+
+## Sharing a funnel
+
+Funnels can be shared with your teams on dashboards to analyze conversion alongside other telemetry metrics, or in a [Notebook][4] to be used for reporting.
+
+Note that you have a few options when it comes to sharing:
+
+- Share the entire visualization 
+
+  [ ADD IMAGE TK ]
+
+- Share individual widgets
+
+{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-share-individual-widgets-1.mp4" alt=alt="Share individual widgets" video="true" >}}
```

Comment:
```suggestion
{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-share-individual-widgets-1.mp4" alt=alt="Share individual widgets" video="true" >}}
```
I think this is what's causing the positional arguments error

Reply:
good find :D